src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,153 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //----------------------------------------------------------------------------
+  // Container
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(500px);
+    width: 100%;
+    overflow-x: auto;
+    padding-top: $spacing-01; // allow space for focus styles
+  }
+
+  //----------------------------------------------------------------------------
+  // Table title text
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
+    padding: rem(22px) 0 0 $spacing-05;
+  }
+
+  .#{$prefix}--data-table-v2-title {
+    @include type-style('productive-heading-03');
+    color: $text-01;
+  }
+
+  .#{$prefix}--data-table-v2-optional {
+    @include type-style('body-short-01');
+    color: $text-02;
+  }
+
+  //----------------------------------------------------------------------------
+  // Data table
+  //----------------------------------------------------------------------------
+  .#{$prefix}--data-table-v2 {
+    border-collapse: collapse;
+    border-spacing: 0;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 thead {
+    @include type-style('heading-01');
+    background-color: $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2 tbody {
+    @include type-style('body-short-01');
+    background-color: $ui-01;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 tr {
+    border: none;
+    height: $layout-04;
+    width: 100%;
+  }
+
+  .#{$prefix}--data-table-v2 tbody tr:hover {
+    background: $hover-field;
+  }
+
+  .#{$prefix}--data-table-v2 tbody tr:hover td {
+    color: $text-01;
+    background: $hover-field;
+    border-bottom: 1px solid $hover-field;
+    border-top: 1px solid $hover-field;
+  }
+
+  .#{$prefix}--data-table-v2 th,
+  .#{$prefix}--data-table-v2 td {
+    padding-left: $spacing-04;
+    padding-right: $spacing-04;
+    vertical-align: top;
+    text-align: left;
+  }
+
+  .#{$prefix}--data-table-v2 th {
+    color: $text-01;
+    background-color: $ui-03;
+    border-top: 1px solid $ui-03;
+    border-bottom: 1px solid $ui-03;
+  }
+
+  .#{$prefix}--data-table-v2 th:first-of-type:not(.#{$prefix}--table-column-checkbox) {
+    padding-left: $spacing-05;
+  }
+
+  .#{$prefix}--data-table-v2 .#{$prefix}--table-header-label {
```

Comment:
Is this a new selector for table headings?

Reply:
This is an existing selector.

Reply:
Cool! Mind pointing me to line in the sass file? Having a hard time finding it 🤔 Will add it to the docs in #2162!

Reply:
It doesn't seem to be used in the sass for v9, but it's in the markup here https://github.com/IBM/carbon-components/blob/7dfc4617e810d083fb5f15b9fdb3f5727ca4a0ab/src/components/data-table-v2/data-table-v2.hbs#L71
https://github.com/IBM/carbon-components/blob/7dfc4617e810d083fb5f15b9fdb3f5727ca4a0ab/src/components/data-table-v2/data-table-v2.hbs#L80

Any headers that are sortable or not small.

Reply:
Yeah, I think it comes from editing work that never got GA'd